main.go

```diff
@@ -88,7 +88,7 @@ func main() {
 	// set up signals so we handle the first shutdown signal gracefully
 	stopCh := signals.SetupSignalHandler()
 
-	httpServer := startHttpServer()
+	httpServer := starthttpServer()
```

Comment:
hmm... this seems odd to me that the linter would want lower case. According to [guidelines](https://github.com/golang/go/wiki/CodeReviewComments#initialisms), protocols et al should be capitalized, so `startHTTPServer`.

Reply:
Correct